src/components/table/CustomToolbar.tsx

```diff
@@ -0,0 +1,16 @@
+import { GridToolbarContainer, GridToolbarExport } from '@mui/x-data-grid';
+
+const CustomToolbar = () => {
+    return (
+        <GridToolbarContainer
+            sx={{
+                display: 'flex',
```

Comment:
I think GridToolbarContainer is already a flex container.

Reply:
Resolved